test/data/validation/destinations/mp.json

```diff
@@ -611,6 +620,147 @@
       "trackCategorizedPages": false,
       "trackNamedPages": false,
       "sourceName": "AWS",
+      "sessionReplayPercentage": {
+        "web": "1"
+      },
+      "crossSubdomainCookie": true,
```

Comment:
try to limit the test case to required scenario only with mandatory ones - no need to repeat all the stuff in each test case.
Or plan a single test case that covers multiple scenarios which makes the larger test case valid

Reply:
done